dataland-frontend/src/components/forms/parts/kpiSelection/AssuranceFormField.vue

```diff
@@ -4,14 +4,11 @@
   <FormKit name="assurance" type="group">
     <!-- Level of assurance -->
     <div class="form-field">
-      <UploadFormHeader
-        :label="euTaxonomyKpiNameMappings.assurance ?? ''"
-        :description="euTaxonomyKpiInfoMappings.assurance ?? ''"
-        :is-required="true"
```

Comment:
manchmal isRequired, manchmal :required="true", manchmal is-required="true", funktioniert das alles gleich?
dann würde ich das einheitlich machen.